CHANGELOG/CHANGELOG-1.18.md

```diff
@@ -15,11 +15,11 @@ When cutting a new release, update the `unreleased` heading to the tag being gen
 
 ## unreleased
 
+* [CHANGE] Update cass-operator to v1.22.0
+
```

Comment:
[question] What's the impact of having the outdated cass-operator image in v1.18.0? Should we prioritize a v1.18.1 release?

Reply:
We don't expose any new feature of 1.22 in k8ssandra-operator, so probably very minor effect?